src/streaming/MediaPlayerEvents.js

```diff
@@ -105,6 +105,10 @@ class MediaPlayerEvents extends EventsBase {
          * @event MediaPlayerEvents#PERIOD_SWITCH_STARTED
          */
         this.PERIOD_SWITCH_STARTED = 'streamswitchstarted';
+
+        this.QUALITY_CHANGE_START = 'qualityChangeStart';
```

Comment:
> The convention for events that go to the client is to be lower case - it came in in 2.0 iirc.

@LloydW93 Will Law pointed out the string names of the events.  Some are all lower case and some camel cased.  Is this what you are referring to?   If so this is just and oversight and does not signify external or internal client.  See more info in this new issue https://github.com/Dash-Industry-Forum/dash.js/issues/1435